### DISCUSSION TITLE
阿里云OSS前端报错但实际上传成功

### DISCUSSION THREAD

**Discussion by wozsun** [AlistGo/alist#4806] [Q&A] [answered]
使用阿里云OSS时，**概率**出现Cannot read properties of undefined (reading 'status')，
<img width="1614" alt="image" src="https://github.com/alist-org/alist/assets/80951424/bab256ec-34a2-43de-a10c-0f6bd66a936b">
<img width="499" alt="image" src="https://github.com/alist-org/alist/assets/80951424/63c27c14-9386-42ca-bec1-2e456576805b">
<img width="498" alt="image" src="https://github.com/alist-org/alist/assets/80951424/b6b1a1e7-d5b2-4b73-a7ab-abd7f00500b0">
但是检查发现实际上传成功了并且后端没有报错
<img width="712" alt="image" src="https://github.com/alist-org/alist/assets/80951424/93157cdb-4edf-4bd6-a84a-e074f9a6c449">
这个应该怎么解决？

**Comment by xhofe** [comment]
F12看一下网络请求 可能是nginx超时了

**Reply by wozsun** [reply]
<img width="1615" alt="image" src="https://github.com/alist-org/alist/assets/80951424/451674d3-fcf5-4f00-b17d-ae9afcb9fbee">
<img width="1637" alt="image" src="https://github.com/alist-org/alist/assets/80951424/12fd01f1-284d-429a-9f1b-5b0da83fcba6">

**Reply by xhofe** [reply]
试下直接在源站上传呢？不使用nginx反代的情况下

**Reply by wozsun** [reply, marked as answer]
找到问题了，不是nginx，是我CDN 10秒就关闭回源链接（epoll_wait() reported that client prematurely closed connection, so upstream connection is closed too while sending request to upstream）导致的出错:cry:不好意思麻烦了:pray:
